pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -2,7 +2,7 @@
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
-, k3sVersion ? null
+, k3sVersion ? "1.25.6-k3s1"
```

Comment:
Would `k3s.version` work too?

Reply:
erm, yeah I guess we could use that but there's no guarantee that k3d version X from 2023-03-17 works with new version of k3s version Y from 2023-04-20 for example

It also might give the false impression that k3d is using the k3s from nixpkgs when it's actually fetching a remote container image

Reply:
my policy so far when updating (previously) has been to just run the upstream container for k3d version X and grab the value they have and use that

Reply:
If it is going to use this hardcoded from somewhere else it is important to at least having a comment referencing the source, ideally of a commit and line. Then, makes sense to add this information to the update script, or the bot will get it wrong.

Reply:
> my policy so far when updating (previously) has been to just run the upstream container for k3d version X and grab the value they have and use that

If the source version can be referenced, this information can be used for the update script, so the bot can properly file a PR for this package, and your workflow can be to just review/approve the PR (using nixpkgs-review). Extra work only being necessary when update script breaks. Also with added bonus of making the update process explicit.

Reply:
Update script is not required to proceed with review. Just desirable.
But the comment pointing to source of information is required.

Reply:
yeah I can put in an update script to keep it inline with upstream

Reply:
Can you update comment so we merge this as it is and then later you file a new PR for update script?

Reply:
@06kellyjac Can you just add the comment and we be done with this?